doc/md/examples/grammar.txt

```diff
@@ -141,7 +141,7 @@
 <exp_obj> ::= 
     '{' <list(<exp_field>, ';')> '}'
     '{' <exp_post> 'and' <exp_post> ('and' <exp_post>)* '}'
-    '{' <exp_field> (';' <exp_field>)* 'in' <exp_post> ('and' <exp_post>)* '}'
+    '{' <exp_post> ('and' <exp_post>)* '|' <list(<exp_field>, ';')> '}'
```

Comment:
Compared to the keyword `in`, using the vertical bar appears more natural to me, especially if I read it as "such that", or "with".

But then  I wonder, would the word `with` be even better?

Currently, the role of `|` in Motoko is limited, but it still does convey something about "or" to me when I read it as a word, which is not appropriate here.

Reply:
Agreed with @matthewhammer. :)